tests/queries/0_stateless/01525_select_with_offset_fetch_clause.reference

```diff
@@ -3,7 +3,7 @@
 5
 1	1
 2	1
-3	4
+3	3
```

Comment:
Is it because of unspecified "tails" handling?

Reply:
Yep, likely some other tests will also have this issue